src/zino/tasks/juniperalarmtask.py

```diff
@@ -40,10 +46,14 @@ async def _get_juniper_alarms(self):
 
         if type(yellow_alarm_count) is not int or type(red_alarm_count) is not int:
             _logger.error(
-                "Device %s returns alarm count not of type int, type yellow alarm count: %s, type red alarm count: %s",
+                "Device %s returns alarm count not of type int. "
+                "Yellow alarm count: type %s, value %s. "
+                "Red alarm count: type %s, value %s.",
                 self.device.name,
                 type(yellow_alarm_count),
+                yellow_alarm_count,
                 type(red_alarm_count),
+                red_alarm_count,
```

Comment:
Is the alarm count value relevant in an error message regarding the typing itself?

Reply:
Yes, since @lunkwill42 explicitly stated that he is interested in that information

Reply:
It could easily be moved to a separate debug statement (which might be more befitting).  But one thing I would like to suggest here is to use `%r` for the values, not `%s`, as that will usually be a more meaningful representation of an unexpected value.